File: JS/CountCommasInRange.test.js
/**
 * @version JAVASCRIPT ECMAScript 6 
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
import TEST from 'tape'
import { countCommas } from './CountCommasInRange.js'

TEST('Starting CountCommasInRange test...', (t) => {
    //Test 1
    let want = 3
    let got = countCommas( 1002 )
    t.assert( want === got, "Expect: "+want)
    //Test 2
    want = 0
    got = countCommas( 998 )
    t.assert( want === got, "Expect: "+want)

    t.end()
})

File: JAVA/CountCommasInRangeTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class CountCommasInRangeTest {

    @Test
	public void testCountCommasInRangeTest() throws Exception {
        //Test 1
        long want = 3;
        long got =  CountCommasInRange.countCommas( 1002);
        assertEquals(want, got);
        //Test 2
        want = 0;
        got =  CountCommasInRange.countCommas( 998 );
        assertEquals(want, got);
	}

}
